wordpress/user.py

```diff
@@ -0,0 +1,41 @@
+import wordpress
+from wordpress.client import APIClient
+
+
+class WPUser:
+    email = None
+    roles = None
+    user_id = None
+    
+    def __init__(self, client=APIClient(), logger=wordpress.logger):
+        self.logger = logger
+        self.roles = set()
+        self.client = client
+
+
+    def fetch(self, email):
```

Comment:
I see that not having the user id persisted in the DB is making this rather inefficient. And I see this will scale quickly as we implement more use cases. We need to start thinking about persistence.

Reply:
You mean saving the WpUsers with id+email in the db for future usercases? it makes sense. For the moment since this is only trigerred when new wordpress user Ids are created (new Woocomerce customers), this use case won't reuse any previous user_id.

Reply:
yes, we can still afford it but it is very clunky to search the user by email to then fetch it again when its id probably comes within the new customer webhook's body.

Reply:
I agree.